src/axom/primal/geometry/BezierCurve.hpp

```diff
@@ -0,0 +1,335 @@
+// Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*!
+ * \file BezierCurve.hpp
+ *
+ * \brief A BezierCurve primitive
+ */
+
+#ifndef PRIMAL_BEZIERCURVE_HPP_
+#define PRIMAL_BEZIERCURVE_HPP_
+
+#include "axom/core.hpp"
+#include "axom/slic.hpp"
+
+#include "axom/primal/geometry/NumericArray.hpp"
+#include "axom/primal/geometry/Point.hpp"
+#include "axom/primal/geometry/Vector.hpp"
+#include "axom/primal/geometry/Segment.hpp"
+#include "axom/primal/geometry/BoundingBox.hpp"
+#include "axom/primal/geometry/OrientedBoundingBox.hpp"
+
+#include "axom/primal/operators/squared_distance.hpp"
+
+#include "fmt/fmt.hpp"
```

Comment:
Is this header needed here? It doesn't look like `fmt` is being used anywhere. Please, remove.